src/lib.rs

```diff
@@ -130,40 +135,48 @@ unsafe fn console_on_any(fds: &[STD_HANDLE]) -> bool {
 
 /// Returns true if there is an MSYS tty on the given handle.
 #[cfg(windows)]
-unsafe fn msys_tty_on(fd: STD_HANDLE) -> bool {
+fn msys_tty_on(handle: HANDLE) -> bool {
```

Comment:
The I/O safety convention is that any function taking a raw `HANDLE` should be marked `unsafe`.